quarkchain/cluster/shard_state.py

```diff
@@ -1562,7 +1562,7 @@ def get_transaction_list_by_address(
             for orderable_tx in self.tx_queue.txs:
                 tx = orderable_tx.tx
                 # TODO: could also show incoming pending tx
-                if Address(tx.sender, tx.from_full_shard_key) == address and (
```

Comment:
we may also want to show it if the tx is **to** that lookup address 
`if tx.sender == address.recipient or tx.to == address.recipient xxx`